sdn/examples/local-ni-multiple-ports/README.md

```diff
@@ -0,0 +1,211 @@
+to determine port connectivity status.
+
+Lastly, port forwarding is configured to enable accessing the app from outside.
+However, this is limited to `eth0`, `eth1` and `eth2` using a shared label `portfwd`
+(purely for example purposes).
+
+Here is a diagram depicting the network topology:
+
+```text
+                         +-------------------+
+              ---------->|    eth0 (mgmt)    |---------------
+              |          |  (DHCP, portfwd)  |              |
+              |          +-------------------+              |
+              |                                             |
++-----+   +----------+   +-------------------+              |
+| app |-->| Local NI |-->| eth1 (app-shared) |              |
++-----+   +----------+   | (static, portfwd) |              |
+              |          +-------------------+              |
+              |                                             |
+              |          +-------------------+          +--------+  +------------+
+              ---------->| eth2 (app-shared) |----------| router |--| httpserver |
+              |          |  (DHCP, portfwd)  |          +--------+  +------------+
+              |          +-------------------+               |
+              |                                              |
+              |          +-------------------+               |
+              ---------->|    eth3 (mgmt)    |----------------
+                         | (static IP conf)  |
+                         +-------------------+
+```
+
+Run the example with:
+
+```shell
+make clean && make build-tests
+./eden config add default
+./eden config set default --key sdn.disable --value false
+./eden setup --eve-bootstrap-file $(pwd)/sdn/examples/local-ni-multiple-ports/device-config.json
+./eden start --sdn-network-model $(pwd)/sdn/examples/local-ni-multiple-ports/network-model.json
+./eden eve onboard
+./eden controller edge-node set-config --file $(pwd)/sdn/examples/local-ni-multiple-ports/device-config.json
+```
+
+Once deployed, check the routing table of the network instance:
+
+```shell
+./eden eve ssh
+eve$ ip route show table 801
+default via 172.22.10.1 dev eth0 proto static
+unreachable default proto static metric 4294967295
+10.40.40.0/24 dev eth3 proto static scope link src 10.40.40.30 metric 1009
+10.50.0.0/24 dev bn1 proto static scope link src 10.50.0.1
+10.88.88.0/24 via 172.22.10.1 dev eth0 proto static
+172.22.10.0/24 dev eth0 proto static scope link src 172.22.10.13 metric 1011
+172.28.20.0/24 dev eth1 proto static scope link src 172.28.20.10 metric 1008
+192.168.30.0/24 dev eth2 proto static scope link src 192.168.30.20 metric 1010
+```
+
+Notice that `eth0` is selected for both the default route and the HTTP server route.
+
+Login to the application and try to access something in the Internet (e.g. 8.8.8.8)
+and the HTTP server:
+
+```shell
+eve$ eve attach-app-console 3599588a-17d3-4d02-aae1-bcefe3706cfd.1.1/cons
```

Comment:
this uuid `3599588a-17d3-4d02-aae1-bcefe3706cfd.1.1/cons` is hardcoded, right?

Reply:
yes, hardcoded in the device config of the example